mcp/mcp-cli/src/main/java/software/amazon/smithy/java/mcp/cli/McpRegistry.java

```diff
@@ -0,0 +1,12 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package software.amazon.smithy.java.mcp.cli;
+
+import picocli.CommandLine;
+
+@CommandLine.Command(name = "mcp-registry", versionProvider = VersionProvider.class, mixinStandardHelpOptions = true,
+        scope = CommandLine.ScopeType.INHERIT)
+class McpRegistry {}
```

Comment:
Why did this have to be moved?

Reply:
I didn't want any accidental class initialization before we add system properties. The annotations on McpCli were triggering some.